application/logging_default_config.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
```

Comment:
Please update https://github.com/junhaoliao/iCtrl/blob/5c7ce900c834cbf199dff26f414473056b45d150/publish/ictrl_be.spec#L11 , or this file will not be included in the packaged application.

Reply:
According to [here](https://docs.python.org/3.11/library/logging.config.html#dictionary-schema-details) the only valid value for version is 1, please correct me if I am wrong.

Reply:
Not sure if I understand the last comment. We need to add this yaml config file to the PyInstaller packaging configuration so the packaged application can find this yaml config file, right?